### DISCUSSION TITLE
Where can I see the source code of the untrained model?

### DISCUSSION THREAD

**Discussion by Hyperplane2021** [google-research/text-to-text-transfer-transformer#953] [Q&A] [answered]
This question may be a bit naive, I am a beginner in NLP.

I fully understood the transformer and wrote the demo in pytorch in order to understand T5 better, but I had some difficulties in reading this code.

What is mesh transform and does this code contain the original untrained model?

**Comment by adarob** [comment, marked as answer]
I think the simplest implementation to read and understand is our new jax-based one: https://github.com/google-research/t5x/tree/main/t5x/examples/t5

**Reply by Hyperplane2021** [reply]
Thank you very much for your reply.
I am just getting into deep learning and this is my first learning model. I have read a lot of transformer articles and tutorials in order to learn it.
It is no exaggeration to say that this model was my initiation tutor.
Thanks again.
